tutorials/dist/1_node_classification.py

```diff
@@ -436,4 +436,49 @@ def forward(self, blocks, x):
   ip_addr3
   ip_addr4
 
+Sample neighbors with `GraphBolt`
```

Comment:
Probably want to mention the benefit of using graphbolt and add a link to introduction of graphbolt